### ISSUE TITLE
Safari+Chrome: input[type=search].form-control with results attribute displays results decoration incorrectly

### ISSUE THREAD

**Issue by vinorodrigues** [twbs/bootstrap#19758]
In webkit based browsers an `<input>` tag of `type="search"`, of `class="form-control"` AND results history switched on (`results="x"`) displays the results dropdown incorrectly:
* on Safari on OSX - it seems to cut off (the a magnifier icon) on the left
* on Chrome (49) - it seems to leave a wide space on left

```html
  <input type="search" class="form-control" results="10">
```

I'm aware that TWBS accounts for _selector_::-webkit-search-cancel-button , but I cannot find a reference to _selector_::-webkit-search-results-button

Example in [JSFiddle](https://jsfiddle.net/vinorodrigues/sLgwgcqa/)

**Comment by twbs-lmvtfy** [comment]
Hi @vinorodrigues!

You appear to have posted a live example (https://fiddle.jshell.net/vinorodrigues/sLgwgcqa/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 74, column 1 thru column 73: Attribute ```` results ```` not allowed on element ```` input ```` at this point.
* line 81, column 1 thru column 94: Attribute ```` results ```` not allowed on element ```` input ```` at this point.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by vinorodrigues** [comment]
True - both `results` and `autosave` are not standard HTML5 attributes per-se - but they are part of webkit... used in both Safari and Chrome.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
